geode-core/src/main/java/org/apache/geode/internal/cache/TXRegionState.java

```diff
@@ -319,14 +319,24 @@ void buildMessage(InternalRegion r, TXCommitMessage msg) {
     try {
       if (!r.getScope().isLocal() && !this.entryMods.isEmpty()) {
 
-        msg.startRegion(r, entryMods.size());
+        TXCommitMessage.RegionCommit regionCommit = msg.startRegion(r, entryMods.size());
         Iterator it = this.entryMods.entrySet().iterator();
```

Comment:
Can this iterator be moved down to above the while loop that uses it, to make things a little clearer?